store/mongo/datastore_mongo_test.go

```diff
@@ -96,12 +96,20 @@ func TestUpdateDeviceStatus(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, model.DeviceStatusDisconnected, device.Status)
 
-	err = ds.UpdateDeviceStatus(ctx, tenantID, deviceID, model.DeviceStatusConnected)
+	err = ds.UpsertDeviceStatus(ctx, tenantID, deviceID, model.DeviceStatusConnected)
 	assert.NoError(t, err)
 
 	device, err = ds.GetDevice(ctx, tenantID, deviceID)
 	assert.NoError(t, err)
 	assert.Equal(t, model.DeviceStatusConnected, device.Status)
+
+	const anotherDeviceID = "efgh"
+	err = ds.UpsertDeviceStatus(ctx, tenantID, anotherDeviceID, model.DeviceStatusConnected)
+	assert.NoError(t, err)
+
+	device, err = ds.GetDevice(ctx, tenantID, anotherDeviceID)
+	assert.NoError(t, err)
+	assert.Equal(t, model.DeviceStatusConnected, device.Status)
```

Comment:
maybe another round of Upsert and Get to check that the device id stayed the same but status was updated?

Reply:
Good idea: https://github.com/mendersoftware/deviceconnect/pull/15/commits/ef44cddfa4837c8640448067074a7003c8bd557c